airbyte-webapp/src/views/Connector/ServiceForm/utils.ts

```diff
@@ -10,8 +11,16 @@ export function serverProvidedOauthPaths(connector?: ConnectorDefinitionSpecific
   [key: string]: { path_in_connector_config: string[] };
 } {
   return {
-    ...(connector?.advancedAuth?.oauthConfigSpecification.completeOAuthOutputSpecification?.properties ?? {}),
-    ...(connector?.advancedAuth?.oauthConfigSpecification.completeOAuthServerOutputSpecification?.properties ?? {}),
+    ...((
+      connector?.advancedAuth?.oauthConfigSpecification?.completeOAuthOutputSpecification as
+        | { properties: Record<string, { path_in_connector_config: string[] }> }
+        | undefined
+    )?.properties ?? {}),
+    ...((
+      connector?.advancedAuth?.oauthConfigSpecification?.completeOAuthServerOutputSpecification as
+        | { properties: Record<string, { path_in_connector_config: string[] }> }
+        | undefined
+    )?.properties ?? {}),
```

Comment:
Unsure if this is correct as `completeOAuthServerOutputSpecification` is returned as `unknown`

Reply:
If it's all we need, great! But here's the actual spec: https://github.com/airbytehq/airbyte/blob/ab55a0cb7854b05a27b6475ac16023e503e6a585/airbyte-api/src/main/openapi/config.yaml#L4202-L4211

Reply:
![Screen Shot 2022-05-09 at 12 04 21 PM](https://user-images.githubusercontent.com/4967694/167450815-dab48ad1-b444-472c-9f08-909b71fb3761.png)
![Screen Shot 2022-05-09 at 12 06 19 PM](https://user-images.githubusercontent.com/4967694/167451178-b42cb9ae-9713-442f-9a11-10fb09fda0d1.png)

I think it's not guaranteed what type we actually get. Just that it _at least_ follows this formula. I'll improve the types a touch!